recipes/qtkeychain/all/conanfile.py

```diff
@@ -29,20 +31,33 @@ class PackageConan(ConanFile):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def validate(self):
+        check_min_cppstd(self, 17)
+        qt_options = self.dependencies["qt"].options
+        if self.settings.os in ("Linux", "FreeBSD") and not qt_options.with_dbus:
+            raise ConanInvalidConfiguration(f'{self.ref} requires -o="qt/*:with_dbus=True" in {self.settings.os}')
+
     def requirements(self):
-        self.requires("qt/[>=6.7 <7]", options={"with_dbus": True}, transitive_headers=True, transitive_libs=True)
-        self.requires("dbus/1.15.8", transitive_headers=True, transitive_libs=True)
+        self.requires("qt/[>=6.7 <7]", transitive_headers=True, transitive_libs=True)
+        # self.requires("dbus/1.15.8")
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "SET(CMAKE_CXX_STANDARD 11)", "")
 
     def generate(self):
         tc = CMakeToolchain(self)
-        tc.variables["BUILD_WITH_QT6"] = True
-        # tc.cache_variables["PACKAGE_BUILD_TESTS"] = False
+        tc.cache_variables["BUILD_WITH_QT6"] = True
+        # Settings this to True would need LinguistTools, which requires qttools, gui and widgets
+        # qttools is not built by default in qt package, so we would need to add a check in validate()
+        tc.cache_variables["BUILD_TRANSLATIONS"] = False
```

Comment:
Turned this off by default because it would need some modules that are not currently built by default in Conan, but if support for translations is desired we can add them back, but no binaries will be publish in CCI for this recipe then